package.json

```diff
@@ -86,7 +86,7 @@
     "babel-jest": "^29.0.0",
     "babel-plugin-replace-ts-export-assignment": "^0.0.2",
     "dedent": "^1.5.0",
-    "eslint": "^7.0.0 || ^8.0.0",
+    "eslint": "^7.0.0 || ^8.0.0 || ^9.0.0",
```

Comment:
@SimenB is there some Yarn trick I don't know about to actually select the version to use locally without a git change?

I'm finding that v9 is being installed, which completely breaks my IDE and tests - alternatively, could we remove v9 from this list? CI was running fine with installing `next` without this so I don't know quite why we need it 😅

Reply:
Have opened https://github.com/jest-community/eslint-plugin-jest/pull/1569